src/main/java/core/basesyntax/FruitTransaction.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.List;
+
+public interface FruitTransaction {
+
+    List<String> getDayData();
+
+    void processData(List<String> data);
+
+    String generateReport();
+
+    void writeData(String report);
```

Comment:
I'm not sure, but according to one of the SOLID's principles , it's better to have several short interfaces than one big one.

![image](https://user-images.githubusercontent.com/79330731/198066271-666f8fb5-602a-47b3-97af-149d863a055a.png)